### DISCUSSION TITLE
Scrollbars for large html-tables with RevealJS

### DISCUSSION THREAD

**Discussion by snhansen** [quarto-dev/quarto-cli#5862] [Q&A] [answered]
### Description

Consider the following code:

	---
	title: "Test"
	format: revealjs
	---

	## 1

	```{r}
	library(modelsummary)
	library(dplyr)
	datasummary_skim(select(mtcars, 1:5))
	```

	## 2

	```{r}
	datasummary_skim(mtcars)
	```
The first table automatically gets a horizontal scrollbar since it's too wide. The second table might get one as well but I can't see it since the table is too long. Should a vertical scrollbar not appear automatically here?

I tried

	## 3

	.scrolling {
	  max-height: 50px;
	  overflow-y: auto;
	}

	::: {.scrolling}

	```{r}
	datasummary_skim(mtcars)
	```

	:::

but that doesn't work here.

**Comment by cderv** [comment, marked as answer]
Scrolling is not applied automatically on slide content that does not fit the size. You can add it though, but the option is `scrollable`
See https://quarto.org/docs/presentations/revealjs/index.html#content-overflow

Either for all slides 
````markdown
---
title: "Test"
format: 
  revealjs:
    scrollable: true
---

## 1

```{r}
library(modelsummary)
library(dplyr)
datasummary_skim(select(mtcars, 1:5))
```

## 2

```{r}
datasummary_skim(mtcars)
```
````

or for a specific slide 
````markdown
---
title: "Test"
format: revealjs
---

## 1

```{r}
library(modelsummary)
library(dplyr)
datasummary_skim(select(mtcars, 1:5))
```

## 2 {.scrollable}

```{r}
datasummary_skim(mtcars)
```

````

**Comment by snhansen** [comment]
Completely missed that option. Thanks for the response, @cderv. Much appreciated.
